src/SFML/Graphics/Texture.cpp

```diff
@@ -462,9 +458,8 @@ void Texture::update(const Uint8* pixels, const Vector2u& size, const Vector2u&
 
 
 ////////////////////////////////////////////////////////////
-void Texture::update(const Texture& texture)
+void sf::Texture::update(const Texture& texture)
 {
-    // Update the whole texture
```

Comment:
Again, please don't remove comments in unrelated files. It's outside of the scope of this PR.